frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -49,7 +49,7 @@
         class="notification is-large is-unselectable has-text-centered is-clickable py-1 mb-1"
         @mousedown="globalSearch()"
       >
-        Limited to 10 results per type. Click here to search all integrated GEMs
+        Limited to 50 results (max 10 per type). Click here to search all integrated GEMs
```

Comment:
To me this sentence is a bit messy. Do users need to know that we have a limit on 50 results in total? To me it makes sense to just keep the sentence the way it was before, but maybe there are other opinions on this

Reply:
I agree it's a bit messy, but only saying `max 10 per type` gives the use a false feeling that the limit is set on individual component but it's not. Maybe we can just show `max 10 per type` but add a sentence in the documentation to explain it.

Reply:
How about a shorter version ?
```suggestion
        limited to 10 rows per type; click here to search all integrated GEMs
```

Reply:
Good suggestion! Using `rows` to separate from results. The description of Quick Search are also updated in the Documentation to reflect that the limit of 50 is for all components instead of per type. see d5727ecc.